node/test/unit/tessel.js

```diff
@@ -565,7 +565,20 @@ exports['Tessel.Port.prototype'] = {
     test.done();
   },
 
-  I2C: function(test) {
+  I2CnoArgsAlwaysHasPort: function(test) {
+    test.expect(3);
+
+    var device1 = new this.port.I2C();
```

Comment:
my question is about the `options.address` not the `options.port`. Shouldn't it hit [this line](https://github.com/tessel/t2-firmware/pull/170/files/2e4ad5c6c15ec91d282155e9c9a739cb595724ad#diff-84087272b70af9498872cf6f791003abR793)?